.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
add .vscode folder in .gitignore. This isn't meant to be checked in.
and remove this from git tracking so that we don't see it here